models/experimental/bert/fused_ops/layernorm.py

```diff
@@ -126,13 +123,13 @@ def layernorm_1d_(x, overrideH=None, refx=None, refgamma=None, refbeta=None):
             H_ = overrideH
 
         # first compute the mean (m)
-        means = tensor.reduce(x, RSUM, RW, 1.0 / W)  # -> NCH1
+        means = tensor.sum(x, 3, True, None, None, 1.0 / W)  # -> NCH1
```

Comment:
`sum(x, 3, True, None, None, 1.0 / W)`
This call is a bit hard to decode. Unclear what those true/none/none mean w/o knowing the signature of the function. Should those be kw_only args?

Reply:
yep, I'll change this one